app/models/follow.rb

```diff
@@ -15,6 +15,9 @@
 class Follow < ApplicationRecord
   include Paginable
   include RelationshipCacheable
+  include RateLimitable
+
+  rate_limit by: :account, family: :follows
```

Comment:
This is somewhat difficult to reason about. From what I gather, this means if you try to create more `Follow` objects than the limit with the same `account`, this will raise an error. However, I can see at least two issues with that:
- an accepted follow request counts twice, afaict? once for the request, once for the accept
- the creation of such a `Follow` object might not be the direct result of an action from the rate-limited party: when accepting a follow request, you could get a confusing error because the potential follower is rate-limited

Reply:
Also, it might cause the `Import::RelationshipWorker` and possibly other workers to fail, in a way that will push them back on the queue. Not sure it is what should happen.